### PR TITLE
feat(Components): Added InputTags component

### LINKED ISSUE #52: Add label field
Skjermbilde 2020-09-20 kl  15 35 12

### PR DESCRIPTION
**Pull request by kvokov**
image

closes #52

### CODE REVIEW

File: src/components/InputTags/InputTags.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    if (reactTagsRef && reactTagsRef.current) {
+      reactTagsRef.current.clearInput();
+      onChange([]);
+    }
+  };
+
+  return (
+    <div
+      className={`tabetalt-input-tags__wrapper ${!!prefix && 'has-prefix'}`}
+      css={css(styles)}
```
Comment (cobraz) [inline src/components/InputTags/InputTags.tsx:46]: This seems like something that can be solved with `rx`?

File: src/components/InputTags/styles.ts [hunk 2]
```diff
@@ -0,0 +1,136 @@
+import { ThemeUIStyleObject } from 'theme-ui';
+
+export default {
+  '&.tabetalt-input-tags__wrapper': {
+    position: 'relative',
+    fontFamily: 'body',
```
Comment (cobraz) [inline src/components/InputTags/styles.ts:6]: Don't think you'll have to define a fontFamily for this.

**Review by cobraz — APPROVED** [review]
I'm not sure about the styling. Let's ask @AnneMatilde to review.

**Comment by AnneMatilde** [comment]
I think this looks great 😊

File: src/components/InputTags/InputTags.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+  return (
+    <InputTags
+      tags={value}
+      suggestions={suggestions}
+      onChange={setValue}
+      {...rest}
+    />
+  );
+};
```
Comment (TanyaDolgopolova) [inline src/components/InputTags/InputTags.stories.tsx:30]: Maybe this code should be moved to a separate file?

**Review by TanyaDolgopolova — APPROVED** [review]

**Review by cobraz — APPROVED** [review]
LGTM 🎉

**Pull request merged**